js/packages/components/onboarding/CreateAccount.tsx

```diff
@@ -163,7 +165,9 @@ const CreateAccountBody = () => {
 					loop={false}
 					onAnimationFinish={async () => {
 						Vibration.vibrate(500)
-						await rnutil.checkPermissions('p2p')
+						Platform.OS === 'web'
+							? navigate('Onboarding.SetupFinished')
```

Comment:
Out of curiosity, how do we navigate to `"Onboarding.SetupFinished"` on mobile?

Reply:
In the checkPermissions function